TypeCobol.Test/Parser/Performance/Performance.cs

```diff
@@ -213,7 +223,65 @@ public void Part3_Incremental_Cobol85_DeepVariables()
         //[Ignore]
         public void Part3_Incremental_TC_DeepTypes()
         {
-            IncrementalPerformance2(DeepTypes, 20692, "                                                                                ");
+            // Insert blank line right before PROCEDURE DIVISION
+            var rangeUpdate = new RangeUpdate(20692, 0, 20692, 0, Environment.NewLine + "                                                                                ");
+            IncrementalPerformance2(DeepTypes, rangeUpdate);
+        }
+
+        [TestMethod]
+        [TestCategory("Performance")]
+        [TestProperty("Time", "long")]
+        //[Ignore]
+        public void Part4_Incremental_LargeFile_LineUpdateAtBeginning()
+        {
+            /*
+             * Line 352, we are renaming 88-level 'CEA-TP-RETURN-DUPKEY' to 'CEA-TP-RETURN-DUPLICATE-KEY'
+             * and then readjust the end of the line to match 80 chars.
+             */
+            var rename = new RangeUpdate(351, 34, 351, 34, "LICATE-");
+            var adjust = new RangeUpdate(351, 49, 351, 56, string.Empty);
+            IncrementalPerformance2(LargeFile, rename, adjust);
```

Comment:
To be equivalent of others tests, I'd rather create only 1 RangeUpdate.
The current code gives me an error.
I'd prefter to create a RangeUpdate that doesn't insert new characters to be sure that we are always in an update situation.
```suggestion
/*
             * Line 352, we are renaming 88-level 'CEA-TP-RETURN-DUPKEY' to 'CEA-TP-RETURN-DUP-EY'
             */
            var rename = new RangeUpdate(351, 34, 351, 35, "-");
            IncrementalPerformance2(LargeFile, rename);
```